server/service/src/sync/translations/store.rs

```diff
@@ -52,26 +55,50 @@ impl SyncTranslation for StoreTranslation {
             return Ok(None);
         }
 
-        let result = StoreRow {
+        // Store
+        let store = StoreRow {
             id: data.id,
             name_id: data.name_id,
             code: data.code,
             site_id: data.site_id,
             logo: data.logo,
         };
 
-        Ok(Some(IntegrationRecords::from_upsert(
-            PullUpsertRecord::Store(result),
-        )))
+        // Store Tags
+        let tags = match data.tags {
+            Some(tags) => tags,
+            None => return Ok(None),
```

Comment:
store with not tags won't be integrated ?

i think it's good to have a test for this, with store with tags = ""